src/content/docs/en/upgrade-astro.mdx

```diff
@@ -180,3 +180,22 @@ A major release may also include some non-breaking changes and improvements that
   Once these features are out of their experimental period, they will follow the normal semantic versioning contract.
 
 - **Improvements to the documentation** (e.g. reference and error messages). They are built from source for the `docs` repository. This allows Astro to quickly update docs fixes and improvements in the cases where documentation source content is stored in the main `astro` repository.
+
+### Node.js support and upgrade policies
+
+#### Support
+
+- Astro supports the **latest _extended_ LTS** versions of Node.js.
+- Astro can support odd versions of Node.js.
+
+#### Upgrade
+
+- Odd versions of Node.js can be deprecated and/or dropped when the next even version of Node.js published. This change can occur in a **minor** release of Astro, after an arbitrary period of time decided by the Astro Core team.
+- Upgrade of the minimum **_extended_ LTS** version of Node.js can occur in a **minor** release of Astro.
+  - If a security flaw in Node.js that **affects Astro** is disclosed and fixed, the Core team can bump the minimum version of the **_extended_ LTS** in a **patch** release.
+- Upgrade of minor or major versions of Node.js - **not** extended LTS - occurs only in major versions of Astro.
+  - If a security flaw in Node.js that **affects Astro** is disclosed and fixed, the Core team can bump the minimum version in a **minor** release.
+
+### Extended maintenance
+ 
+The Core team will provide extended maintenance for security fixes, only to the previous major of the current release. This means that of the **current major** is `v4.*`, the Core team will backport security fixes on for `v3.*` versions. 
```

Comment:
```suggestion
The Core team will provide extended maintenance **for security fixes only** for one previous major version. This means that if the **current major** is `v4.*`, the Core team will back port security fixes and issue a new `v3.*` release. 
```
Just checking, but I think last time we issued a new v3.x release, not backported compatibility for everyone on every single previous v3.* version? I would frame it more like above, where "we'll issue an updated v3 version you can update to" and not "you can stay on whichever version of 3 you want"

Reply:
That's a good point, thank you for the clarification